patches/server/0465-Add-additional-open-container-api-to-HumanEntity.patch

```diff
@@ -3,47 +3,155 @@ From: JRoy <joshroy126@gmail.com>
 Date: Wed, 26 Aug 2020 02:12:31 -0400
 Subject: [PATCH] Add additional open container api to HumanEntity
 
+Expand existing methods to also allow for title overrides
 
+Co-authored-by: Lukas Planz <lukas.planz@web.de>
+
+diff --git a/src/main/java/net/minecraft/server/level/ServerPlayer.java b/src/main/java/net/minecraft/server/level/ServerPlayer.java
+index e087b62f88157d57eb037feb0d39c076b286f755..4b9319efaca5de6e60f82e35d643bc72b217c9e2 100644
+--- a/src/main/java/net/minecraft/server/level/ServerPlayer.java
++++ b/src/main/java/net/minecraft/server/level/ServerPlayer.java
+@@ -1502,7 +1502,7 @@ public class ServerPlayer extends Player {
+     }
+ 
+     @Override
+-    public OptionalInt openMenu(@Nullable MenuProvider factory) {
++    public OptionalInt openMenu(@Nullable MenuProvider factory, @Nullable net.kyori.adventure.text.Component defaultTitleOverride) { // Paper - overridable title
```

Comment:
I think instead of this diff, you can just create new instances of SimpleMenuProvider and pass in the new Component title. That way it avoids all the diff in this methods and doesn't add yet another local var that is related to a title.